mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/TiltaksgjennomforingRepository.kt

```diff
@@ -24,7 +25,10 @@ import java.util.*
 class TiltaksgjennomforingRepository(private val db: Database) {
     private val logger = LoggerFactory.getLogger(javaClass)
 
-    fun upsert(tiltaksgjennomforing: TiltaksgjennomforingDbo) {
+    fun upsert(tiltaksgjennomforing: TiltaksgjennomforingDbo) =
+        db.transaction { upsert(tiltaksgjennomforing, it) }
+
+    fun upsert(tiltaksgjennomforing: TiltaksgjennomforingDbo, tx: TransactionalSession) {
```

Comment:
Hva er grunnen til at vi har `TransactionalSession` her, men `Session` i `AvtaleNotatRepository` (filen endret over)?

Reply:
Tror vi godt kan bruke Session begge steder. I praksis kommer man nok til å sende inn en TransactionalSession alltid, men mindre man lager en `db.withSession` metode og gjør sånn her i stedet
```
fun foo() = db.withSession { tx -> foo(tx) }
```
På en metode man ønsker å kunne kalle i en transaskjson. Men da kan man likesågodt bruke db.transaction. Klarer ikke helt å se for meg hvorfor man ikke bare kan gjøre det 🤔 

Uansett så har det ikke så mye å si, de er jo veldig like. Så vi kan bare bruke Session.

Reply:
Yes! Var mest av nysgjerrighet om forskjellen var bevisst eller litt tilfeldig :ok_hand: